src/rdkafka_request.c

```diff
@@ -460,6 +460,59 @@ int rd_kafka_buf_write_topic_partitions(
 }
 
 
+/**
+ * @brief Read current leader from \p rkbuf.
+ *
+ * @param rkbuf buffer to read from
+ * @param CurrentLeader is the CurrentLeader to populate.
+ * @returns 1 on success, else -1 on parse error.
```

Comment:
better standardizing on return as returns is an alias for that
```suggestion
 * @return 1 on success, else -1 on parse error.
```

Reply:
Updated.